packages/now-next/test/fixtures/16-base-path/now.json

```diff
@@ -0,0 +1,20 @@
+{
+  "version": 2,
+  "builds": [{ "src": "package.json", "use": "@now/next" }],
+  "probes": [
+    {
+      "path": "/docs/_next/static/testing-build-id/pages/index.js",
+      "responseHeaders": {
+        "cache-control": "public,max-age=31536000,immutable"
+      }
+    },
+    {
+      "path": "/docs/",
```

Comment:
Test the non-slash version of this per my above suggestion.

Reply:
Added additional check